proto/wallet/wallet.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+package proto.wallet;
+option go_package = "github.com/textileio/go-auctions-client/gen/wallet;wallet";
+
+message ProposalSigningRequest {
+	string auth_token = 1;
+
+	string filecoin_deal_protocol = 2;
+	bytes payload = 3;
+}
+
+message ProposalSigningResponse {
```

Comment:
Not sure if it's worth adding a version here too?

Reply:
~Replied above.